__tests__/lotto.test.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable */
+import Lotto from '../src/domain/lotto.js';
+import { ERROR_MESSAGES } from '../src/constant/messages.js';
+
+describe('로또 기능 테스트', () => {
+  test.each([[[1, 2, 3, 4, 5]], [[1, 2, 3, 4, 5, 6, 7]]])('로또 번호가 6개가 아닌 경우 에러를 반환', (lottoNumbers) => {
+    expect(() => {
+      new Lotto(lottoNumbers);
+    }).toThrow(ERROR_MESSAGES.incorrect_length);
+  });
+
+  test('로또 번호가 6개인 경우 정상 작동', () => {
```

Comment:
그나저나 요거는 저번에 못드린 피드백이긴 한데요!
테스트 코드에 대한 명세는 왠만하면 완성형 문장으로 쓰는 것 같아요.

```suggestion
  test('입력 가능한 로또 번호는 6개이다.', () => {
```
이런식으로요?
정반적으로 약간 테스트코드가 일종의 기능명세서라고 생각하고 작성해봐주심 좋아요ㅎㅎ
그래서 저는 요구사항들을 한줄한줄 그대로 테스트 명세로 쓰곤 합니다ㅎㅎ 따로 요구사항 명세 파일이 필요 없도록요!